packages/ocaml-r/ocaml-r.0.4.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "Objective Caml bindings for the R interpreter"
+description: """
+
+OCaml-R is a library that can be used to construct R values in memory,
+convert them to OCaml values, and build clean wrappers to R
+functions. It provide a simple means to develop bindings to any R
+package."""
+maintainer: ["philippe.veber@gmail.com"]
+authors: ["Guillaume Yzyquel" "Maxence Guesdon" "Philippe Veber"]
+license: "GPL3"
+tags: ["R" "statistics"]
+homepage: "https://github.com/pveber/ocaml-r/"
+doc: "https://pveber.github.io/ocaml-r/ocaml-r/index.html"
+bug-reports: "https://github.com/pveber/ocaml-r/issues"
+depends: [
```

Comment:
```suggestion
depends: [
  "ocaml" {>= "4.08"}
```

This will solve the failure on `4.07`:
```
#=== ERROR while compiling ocaml-r.0.4.0 ======================================#
# context              2.0.7 | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(https://github.com/pveber/ocaml-r/releases/download/v0.4.0/ocaml-r-v0.4.0.tbz)
# path                 ~/.opam/4.07/.opam-switch/build/ocaml-r.0.4.0
# command              ~/.opam/4.07/bin/dune build -p ocaml-r -j 72 @install
# exit-code            1
# env-file             ~/.opam/log/ocaml-r-23-d2543e.env
# output-file          ~/.opam/log/ocaml-r-23-d2543e.out
### output ###
#       ocamlc lib/base/.OCamlR_base.objs/byte/oCamlR_base.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.07/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/base/.OCamlR_base.objs/byte -I lib/.OCamlR.objs/native -I lib/.OCamlR.objs/public_cmi -I lib/interpreter/.OCamlR_interpreter.objs/byte -I lib/interpreter/.OCamlR_interpreter.objs/native -I lib/wraputils/.OCamlR_wraputils.objs/byte -I lib/wraputils/.OCamlR_wraputils.objs/native -intf-suffix .ml -no-alias-deps -o lib/base/.OCamlR_base.objs/byte/oCamlR_base.cmo -c -impl lib/base/OCamlR_base.ml)
# File "lib/base/OCamlR_base.ml", line 176, characters 7-17:
# Error: Unbound module Option
#       ocamlc lib/stats/.OCamlR_stats.objs/byte/oCamlR_stats.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.07/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/stats/.OCamlR_stats.objs/byte -I lib/.OCamlR.objs/native -I lib/.OCamlR.objs/public_cmi -I lib/base/.OCamlR_base.objs/byte -I lib/base/.OCamlR_base.objs/native -I lib/interpreter/.OCamlR_interpreter.objs/byte -I lib/interpreter/.OCamlR_interpreter.objs/native -I lib/wraputils/.OCamlR_wraputils.objs/byte -I lib/wraputils/.OCamlR_wraputils.objs/native -intf-suffix .ml -no-alias-deps -o lib/stats/.OCamlR_stats.objs/byte/oCamlR_stats.cmo -c -impl lib/stats/OCamlR_stats.ml)
# File "lib/stats/OCamlR_stats.ml", line 63, characters 7-17:
# Error: Unbound module Option
#     ocamlopt lib/base/.OCamlR_base.objs/native/oCamlR_base.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.07/bin/ocamlopt.opt -w -40 -g -I lib/base/.OCamlR_base.objs/byte -I lib/base/.OCamlR_base.objs/native -I lib/.OCamlR.objs/native -I lib/.OCamlR.objs/public_cmi -I lib/interpreter/.OCamlR_interpreter.objs/byte -I lib/interpreter/.OCamlR_interpreter.objs/native -I lib/wraputils/.OCamlR_wraputils.objs/byte -I lib/wraputils/.OCamlR_wraputils.objs/native -intf-suffix .ml -no-alias-deps -o lib/base/.OCamlR_base.objs/native/oCamlR_base.cmx -c -impl lib/base/OCamlR_base.ml)
# File "lib/base/OCamlR_base.ml", line 176, characters 7-17:
# Error: Unbound module Option

-     ocamlopt lib/base/.OCamlR_base.objs/native/oCamlR_base.{cmx,o} (exit 2)
- (cd _build/default && /home/opam/.opam/4.07/bin/ocamlopt.opt -w -40 -g -I lib/base/.OCamlR_base.objs/byte -I lib/base/.OCamlR_base.objs/native -I lib/.OCamlR.objs/native -I lib/.OCamlR.objs/public_cmi -I lib/interpreter/.OCamlR_interpreter.objs/byte -I lib/interpreter/.OCamlR_interpreter.objs/native -I lib/wraputils/.OCamlR_wraputils.objs/byte -I lib/wraputils/.OCamlR_wraputils.objs/native -intf-suffix .ml -no-alias-deps -o lib/base/.OCamlR_base.objs/native/oCamlR_base.cmx -c -impl lib/base/OCamlR_base.ml)
- File "lib/base/OCamlR_base.ml", line 176, characters 7-17:
- Error: Unbound module Option
```